### ISSUE TITLE
 Uncaught exception: Error: Call to undefined method

### ISSUE THREAD

**Issue by denarie** [matomo-org/plugin-GoogleAnalyticsImporter#44]
When starting the importer I get the following error:

WARNING: matomo/plugins/GoogleAnalyticsImporter/Controller.php(271): Notice - Undefined variable: importStatus - Matomo 3.13.0 - (Module: GoogleAnalyticsImporter, Action: startImport, In CLI mode: false)

**Comment by denarie** [comment]
Sorry, I did not have the newest version of the importer.

I updated the importer and started another import. I don't see anything on the matomo view, even verbose mode does not show any output. I started the job on the command line:

`php ./console googleanalyticsimporter:import-reports --property=UA-XXX --account=XXXX --view=XXX`

And got the following error:
                       
```
  [RuntimeException]   
  Too many arguments.  
```
